lua/wire/stools/expression2.lua

```diff
@@ -268,6 +268,14 @@ if SERVER then
 
 			local filepath = ret[3]
 
+			if ply ~= toent.player then
+				toent.player = ply
+				toent:SetPlayer(ply) -- This function doesn't even exist in the wiki, along with ENT:GetPlayer() and ENT:GetPlayerName() which is used for the chip overlay
```

Comment:
https://github.com/Facepunch/garrysmod/blob/be251723824643351cb88a969818425d1ddf42b3/garrysmod/gamemodes/sandbox/entities/entities/base_gmodentity.lua#L70

Reply:
Yeah that’s how I found it and figured out what it does, just no docs.